### PR TITLE
Fix Travis CI builds as well as update dependencies to latest versions

### PR DESCRIPTION
**Pull request by coltongit**
This PR may be a big one, but all build tests should now pass just fine.

### CODE REVIEW

**Comment by coltongit** [comment]
This PR also requires Node 10+. Builds ran using Node 10/11 should pass OK, as long as saucelabs is configured correctly here

**Review by ljharb — CHANGES_REQUESTED** [review]

**Comment by ljharb** [comment]
Requiring node 10+ is a breaking change that is best avoided, especially for a bundler polyfill like this one.

File: index.js [hunk 1]
```diff
@@ -4,18 +4,18 @@ exports.randomBytes = exports.rng = exports.pseudoRandomBytes = exports.prng = r
 exports.createHash = exports.Hash = require('create-hash')
 exports.createHmac = exports.Hmac = require('create-hmac')
 
-var algos = require('browserify-sign/algos')
-var algoKeys = Object.keys(algos)
-var hashes = ['sha1', 'sha224', 'sha256', 'sha384', 'sha512', 'md5', 'rmd160'].concat(algoKeys)
+const algos = require('browserify-sign/algos')
```
[the remark this answers is not in the document]
Comment (coltongit) [reply index.js:7]: standard is indeed forcing this else the travis builds will fail

**Comment by coltongit** [comment]
Superseding this PR with a new one.

File: index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ljharb) [reply index.js:7]: Seems easier to just repurpose this PR :-)

[the remark this answers is not in the document]
Comment (coltongit) [reply index.js:7]: Too much work put in to not supersede this PR, so better to just supersede it instead.

[the remark this answers is not in the document]
Comment (ljharb) [reply index.js:7]: ok, not sure i understand that - you can just start over and force push to the same branch.

**Comment by coltongit** [comment]
This PR has been superseded by PR #200.

**Pull request closed without merging**